app/src/main/java/de/tum/in/tumcampusapp/database/repository/NetAPIImpl.kt

```diff
@@ -0,0 +1,24 @@
+package de.tum.`in`.tumcampusapp.database.repository
+
+import android.content.Context
+import com.google.gson.GsonBuilder
+import de.tum.`in`.tumcampusapp.api.Helper
+import retrofit2.Retrofit
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory
+import retrofit2.converter.gson.GsonConverterFactory
+
+class NetAPIImpl(val context: Context) : NetAPI {

```

Comment:
Shouldn't this be part of the TUMCabiApi class? It uses the same endpoint?

Reply:
This certainly doesn't fit into the `database` package...

Reply:
aw yeah i guess.